partner_contact_birthdate/__init__.py

```diff
@@ -0,0 +1,4 @@
+# Copyright (C) 2014-2015  Grupo ESOC <www.grupoesoc.es>
+# © 2017-Apertoso N.V. (<http://www.apertoso.be>)
+#  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
```

Comment:
And the license lines should be changed to https:

Reply:
DONE